ui/src/components/VolumeListTable.js

```diff
@@ -290,7 +290,7 @@ function Table({
         query.delete('sort');
         query.delete('desc');
       }
-      history.push(`?${query.toString()}`);
+      history.replace(`?${query.toString()}`);
```

Comment:
I would say to keep `history.push` here too ;)

Reply:
@ChengYanJin No, I didn't miss it :) replacing it here breaks the fix. This useEffect triggers on initial sort and leaving push makes the user clicks two time on "Go back" to go pack to the  previous page.

Reply:
@alexis-ld Indeed. Sorry, I forgot it's in the useEffect. 
Giving my approval!